.github/workflows/checkPot.yml

```
@@ -0,0 +1,52 @@
+name: Check comments for translators
+
+on:
+  push:
+    branches:
+    - master
+    - beta
+    - rc
+    - try-**
+    - release-**
+    - githubActions # To be removed
```

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore

```suggestion
    - 'try-**'
    - githubActions # To be removed
  tags:
    - release-**
```